code/obj/item/clothing/armor.dm

```diff
@@ -406,7 +406,7 @@
 
 /obj/item/clothing/suit/armor/NT
 	name = "armored nanotrasen jacket"
-	desc = "An armored jacket worn by NanoTrasen security commanders."
+	desc = "An armored jacket given by nanotrasen for when their bureaucrats inevitably get shoved into hostile situations."
```

Comment:
This is used for other roles too, would be better to make a variant for inspector

Reply:
The only other thing its used for is a decrepit old nt role that i have never seen used